synop2bufr/__init__.py

```diff
@@ -295,6 +336,23 @@ def parse_synop(message: str, year: int, month: int) -> dict:
         except Exception:
             output['dewpoint_temperature'] = None
 
+    # Verify that the dewpoint temperature is less than or equal to
```

Comment:
Where did this come from? I'm not sure how the dew point compares when the air is supersaturated, maybe just log warning.